src/tools/fuzzing/fuzzing.cpp

```diff
@@ -2468,9 +2468,14 @@ Expression* TranslateToFuzzReader::makeBasicRef(Type type) {
     case HeapType::string:
       return builder.makeStringConst(std::to_string(upTo(1024)));
     case HeapType::stringview_wtf8:
+      return builder.makeStringAs(
+        StringAsWTF8, makeBasicRef(Type(HeapType::string, NonNullable)));
     case HeapType::stringview_wtf16:
+      return builder.makeStringAs(
+        StringAsWTF16, makeBasicRef(Type(HeapType::string, NonNullable)));
     case HeapType::stringview_iter:
-      WASM_UNREACHABLE("TODO: strings");
+      return builder.makeStringAs(
+        StringAsIter, makeBasicRef(Type(HeapType::string, NonNullable)));
```

Comment:
Don't we have separate heap types for all the different stringref views and iterators?

Reply:
Yes, there are three cases here, `stringview_wtf8, wtf16, iter` - ? I'm not sure what else you expect to see, sorry.

Reply:
Oh, I think I was confusing `HeapType::string` for the output type of the expression rather than the input type.